### ISSUE TITLE
First Installation Error

### ISSUE THREAD

**Issue by weedeej** [mail-in-a-box/mailinabox#2447]
On first installation, I everything is smooth until the NextCloud installation. Here's the log

```bash
Mail-in-a-Box Version: v70

Updating system packages...
Installing system packages...
Initializing system random number generator...
Firewall is active and enabled on system startup
Synchronizing state of fail2ban.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable fail2ban
Installing nsd (DNS server)...
Installing Postfix (SMTP server)...
Installing Dovecot (IMAP server)...
Installing OpenDKIM/OpenDMARC...
Installing SpamAssassin...
Installing Nginx (web server)...
Installing Roundcube (webmail)...
Installing Nextcloud (contacts/calendar)...

FAILED: sudo -u www-data php8.0 /usr/local/lib/owncloud/console.php app:enable contacts
-----------------------------------------
Could not download app contacts
-----------------------------------------
```

OS: Ubuntu 22.04

**Comment by zacuke** [comment]
Did you retry the installation? The servers hosting Nextcloud downloads seem to have intermittent issues. 

I have this proposed change, to help log problems of downloading assets during setup #2474 

Although your error appears to be from within a Nextcloud script, it's the same issue creeping up: Nextcloud download servers are intermittent.

**Comment by downtownallday** [comment]
I agree that Nextcloud download servers become unresponsive especially when used frequently in a short time frame, so trying again after a few minutes may fix the issue.

The change proposed of adding a progress indicator to wget won't help in this case because the error occurs during `occ app:enable contacts`, not during wget. This command is causing the local Nextcloud installation to download the contacts app from the Nextcloud app store, and is failing.

As I pointed out in #2319, obtaining the contacts app using wget is doing nothing other than taking disk space. That download is uncompiled (eg. raw .vue files exist) and won't work as an app.

**Comment by zacuke** [comment]
While my change to add a progress indicator doesn't actually fix _anything_, instead the idea is to collect log data to help people see it's the Nextcloud downloads that are failing. 

As this particular issue seems to be a failed download from within a Nextcloud script, is there a verbose option that could be passed to the Nextcloud script to get it to log download progress as well?

Regarding https://github.com/mail-in-a-box/mailinabox/pull/2319 you might consider creating a smaller pull request that only removes the extra downloads that aren't used. That would help clarify this whole situation, and less code change at once improves the odds of the maintainers approving the pull request.

Do you think the mail-in-a-box setup script is triggering some kind of throttling mechanism on Nextcloud's download servers? It would make more sense than trying to say Nextcloud's download servers are flaky, because surely they would have resolved that by now.

**Comment by orlovnv** [comment]
Have the sae issue 
Mail-in-a-Box Version: v71a


Updating system packages...
Installing system packages...
Initializing system random number generator...
Opening alternate SSH port 56777.
Firewall is active and enabled on system startup
Synchronizing state of fail2ban.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable fail2ban
Installing nsd (DNS server)...
Installing Postfix (SMTP server)...
Installing Dovecot (IMAP server)...
Installing OpenDKIM/OpenDMARC...
Installing SpamAssassin...
Installing Nginx (web server)...
Installing Roundcube (webmail)...
Installing Nextcloud (contacts/calendar)...

FAILED: sudo -u www-data php8.0 /usr/local/lib/owncloud/console.php app:enable contacts
-----------------------------------------
Could not download app contacts
-----------------------------------------
